week-4/A-objects-intro/exercise-part-1.js

```diff
@@ -7,3 +7,18 @@ Assign each of them to a separate variable
 
 */
 
+const miles ={
+    ethnicity: "Nigerian",
+    nationality: "British",
+    heigth: "177 KG",
+    weigth: "95 KG",   
+};
+
+const gymStats = {
+    bench: "130 KG",
+    squat: "195 KG",
+    deadlift: "200 KG",
+    overHeadPress: "85 KG"
+};
+
+console.log(miles.nationality+gymStats.bench);
```

Comment:
This line made me curious. Did it work? What happened? JavaScript does often allow to do operations with two values of different types with sometimes peculiar outcomes. Here are some of them if you're interested: https://charlieharvey.org.uk/page/javascript_the_weird_parts